jackbox_patcher/assets/l10n/app_de.arb

```diff
@@ -127,10 +127,10 @@
     "connection_to_server_failed": "Verbindung zum Server fehlgeschlagen",
     "connection_to_main_server_failed": "Aufgrund eines Fehlers konnten die verfügbaren Server nicht abgerufen werden.",
     "quit_while_downloading_title": "Ein Download ist aktiv",
-    "quit_while_downloading_description": "Ein Download ist aktiv, bist du sicher, dass du das Programm schließen möchtest?",
+    "quit_while_downloading_description": "Ein Download ist zurzeit aktiv, bist du sicher, dass du die Anwendung schließen möchtest?",
```

Comment:
"Ein Download ist zurzeit aktiv, bist du sicher, dass du die App schließen möchtest?"

Reply:
Because the App thing got mentioned in a lot of your reviews I would explain it here for them all. Why I think **Anwendung** would be more fitting. As someone who comes from a Software development environment I and a lot of people differentiate them in German. Short App usually means a mobile application. While Program / Application is ment for fully fleged software and Desktop Applications. In the end this probably preference, but if you ask someone in German what an **App** or **Anwendung** is. Most Germans would say that an App is a mobile-app and an Anwendung is a Web-Application or Desktop-Application.

Maybe this helps to explain why I choosed to change it.